client/my-sites/ads/form-settings.jsx

```diff
@@ -381,8 +381,8 @@ const AdsFormSettings = React.createClass( {
 						name="tos"
 						checkedLink={ this.linkState( 'tos' ) }
 						disabled={ this.state.isLoading || 'signed' === this.state.tos } />
-					<span>{ this.props.translate( 'I have read and agree to the {{a}}WordAds Terms of Service{{/a}}.', {
-						components: { a: <a href="https://wordpress.com/tos-wordads/" target="_blank" rel="noopener noreferrer" /> }
+					<span>{ this.props.translate( 'I have read and agree to the {{a}}Automattic Ads Terms of Service{{/a}}.', {
```

Comment:
Alternate string suggestion: 
* `translate( 'I have read and agree to the %sWordAds Terms of Service' )`&mdash; translations: **17**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/ads-tos-link)